### PR TITLE
CB-19874 adding more log to environment validation step to help us de…

### PR DESCRIPTION
**Pull request by doktoric**
…bugging problems

See detailed description in the commit message.

### CODE REVIEW

File: environment/src/main/java/com/sequenceiq/environment/environment/flow/creation/handler/EnvironmentValidationHandler.java [hunk 1]
```diff
@@ -134,19 +136,30 @@ private void validateEnvironment(EnvironmentValidationDto environmentValidationD
         EnvironmentDto environmentDto = environmentValidationDto.getEnvironmentDto();
         RegionWrapper regionWrapper = environment.getRegionWrapper();
         CloudRegions cloudRegions = environmentService.getRegionsByEnvironment(environment);
+        LOGGER.debug("Validate Environment cloud region and location which is {}.", cloudRegions);
         ValidationResult.ValidationResultBuilder validationBuilder = validatorService
                 .validateRegionsAndLocation(regionWrapper.getName(), regionWrapper.getRegions(), environment, cloudRegions);
+        LOGGER.debug("Validate Environment telemetry location.");
```
Comment (topolyai5) [inline environment/src/main/java/com/sequenceiq/environment/environment/flow/creation/handler/EnvironmentValidationHandler.java:142]: maybe the log better inside the validate method

File: environment/src/main/java/com/sequenceiq/environment/environment/validation/validators/EnvironmentNetworkProviderValidator.java [hunk 2]
```diff
@@ -49,8 +49,11 @@ public ValidationResult validate(EnvironmentValidationDto environmentValidationD
         NetworkDto network = environmentDto.getNetwork();
         String cloudPlatform = environmentDto.getCloudPlatform();
         ValidationResultBuilder resultBuilder = ValidationResult.builder();
+        LOGGER.debug("Validate network parameters that cloudplatform specific params are filled up properly.");
         validateNetworkHasTheSamePropertyFilledAsTheDesiredCloudPlatform(network, cloudPlatform, resultBuilder);
+        LOGGER.debug("Validate network that it exist on provider side.");
         validateNetwork(network, cloudPlatform, resultBuilder, environmentValidationDto);
+        LOGGER.debug("Validate security group that it exist on provider side.");
```
Comment (lajosrodek) [inline environment/src/main/java/com/sequenceiq/environment/environment/validation/validators/EnvironmentNetworkProviderValidator.java:56]: Nit: `exist` -> `exists`

**Review by lajosrodek — CHANGES_REQUESTED** [review]
A few small asks.

**Review by topolyai5 — APPROVED** [review]

**Review by lajosrodek — APPROVED** [review]
Thanks, LGTM

**Review by lnardai — APPROVED** [review]

**Pull request merged**